src/Nancy/DefaultRuntimeEnvironmentInformation.cs

```diff
@@ -35,10 +36,20 @@ private static bool GetDebugMode(ITypeCatalog typeCatalog)
             {
                 var assembliesInDebug = typeCatalog
                     .GetTypesAssignableTo<INancyModule>(TypeResolveStrategies.ExcludeNancy)
+#if DOTNET5_4
+                    .Select(x => x.GetTypeInfo().Assembly.GetCustomAttributes(typeof(DebuggableAttribute)))
+                    .Where(x => x.Any());
+
+                    return assembliesInDebug.Any(d => ((DebuggableAttribute)d.First()).IsJITTrackingEnabled);
+#else
                     .Select(x => x.Assembly.GetCustomAttributes(typeof(DebuggableAttribute), true))
                     .Where(x => x.Length != 0);
+                    
+                    return assembliesInDebug.Any(d => ((DebuggableAttribute)d[0]).IsJITTrackingEnabled);

```

Comment:
I had an issue here where the method sig for GetCustomAttributes had changed, we can no longer check for inherited attributes by passing true as the second arg,

This section of code also worried me because there is no way to get the `IsJITTrackingEnabled` yet, and I fear it may need to be scrapped :(.